docs/sources/mimir/release-notes/v2.12.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Grafana Mimir version 2.12 release notes"
+menuTitle: "V2.12 release notes"
+description: "Release notes for Grafana Mimir version 2.12"
+weight: 1000
+---
+
+# Grafana Mimir version 2.12 release notes
+
+Grafana Labs is excited to announce version 2.12 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Added mimir-distroless docker image, built upon the distroless. This improvement is geared towards minimising attack surfaces and mitigating potential CVEs by trimming down the dependencies within the image. After comprehensive testing, we plan to shift from the default docker image to distroless.
+- **Added support to only count series that are considered _active_** through the Cardinality API endpoint `/api/v1/cardinality/label_names` by passing the `count_method` parameter. If set to `active` it counts only series that are considered _active_ according to the `-ingester.active-series-metrics-idle-timeout` flag setting rather than counting all in-memory series.
+- **Added new admin pages that display details about compaction jobs computed from bucket index**. They provide the same output as the `tools/compaction-planner` tool, without the need to provide all necessary configuration via CLI flags, since compactor already has all the required configuration. The new pages can be displayed by the following endpoints:
+  - `GET /compactor/tenants`: displays a web page with the list of tenants that have blocks in the storage configured for the compactor.
+  - `GET /compactor/tenant/{tenant}/planned_jobs`: displays a web page listing planned compaction jobs computed from the bucket index for the given tenant.
+- **The total number of OTLP requests** is tracked by the new `cortex_distributor_otlp_requests_total` metric.
+- **The total number of queries executed for a particular source (ingester vs. store-gateway)** is tracked by the new `cortex_querier_queries_storage_type_total` metric.
+- **The total number of chunks fetched from a store gateway** is tracked by the new `cortex_querier_query_storegateway_chunks_total` metric.
```

Comment:
Interested readers can read all about these metrics in the changelog. Are these note-worthy in the release notes?

Reply:
@pstibrany do you think we should just mention which metrics have been added without describing them, or do you think we should completely remove these entries?

Reply:
It seems to me that new metrics like this are too small and not noteworthy-enough to be in the release notes. But ultimately, it's our PM's decision what to include in the release notes.

Reply:
@zhehao-grafana WDYT about this? Should we mention all these metrics, or should we just leave them in CHANGELOG.md?

Reply:
I will vote to keep them in the changlog

Reply:
I have removed these 2 entries from the release notes. Thank you.